pkg/apis/v1alpha2/aksnodeclass_status.go

```diff
@@ -26,6 +26,7 @@ import (
 const (
 	ConditionTypeImagesReady            = "ImagesReady"
 	ConditionTypeKubernetesVersionReady = "KubernetesVersionReady"
+	ConditionTypeSubnetReady            = "VnetSubnetIDReady"
```

Comment:
Believe you'll want to add this readiness condition to the test default status, even if it's not causing any direct failures:
https://github.com/Azure/karpenter-provider-azure/blob/3352b2d39ddaf6a11406f7beba861b5ba41e70f7/pkg/test/nodeclass.go#L66

Reply:
added to here